### PR TITLE
GDT readme: log level configuration

### PR DESCRIPTION
**Pull request by maksymmalyhin**
#no-changelog

### CODE REVIEW

**Comment by maksymmalyhin** [comment]
cc @mikehaney24

File: GoogleDataTransport/README.md [hunk 1]
```diff
@@ -3,6 +3,29 @@
 This library is for internal Google use only. It allows the logging of data and
 telemetry from Google SDKs.
 
+## Set logging level
+
+### Swift
+
+- Impoort `GoogleDataTransport` module:
```
Comment (paulb777) [inline GoogleDataTransport/README.md:10]: Import

**Review by paulb777 — APPROVED** [review]

**Comment by paulb777** [comment]
Fix check

**Review by samedson — APPROVED** [review]
This is great 👍

**Comment by maksymmalyhin** [comment]
Merging without waiting for checks not relevant for the readme changes.

**Pull request merged**